loops-and-arrays.js

```diff
@@ -22,6 +26,14 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+for (let i = 0; i < numbers.length; i++) {
+   // console.log(numbers[i]);
+    if(numbers[i]%2==0){
```

Comment:
Volgens de aandachtspunten moet je spaties gebruiken tussen je operators

Reply:
Het komt een paar keer terug in je twee files

Reply:
gedaan .